docs/howto/openstack/designate/recordsets.md

```diff
@@ -158,6 +158,9 @@ nuuk.example.com.  3600    IN  AAAA    2001:db8:ffff:ffff:ffff:ffff:ffff:ffff
 ;; MSG SIZE  rcvd: 78
 ```
 
+> For extra resilience, the two name servers in any zone reside in different geographic locations.
+> You may see the location of a name server [here](../../../../reference/designate/).
```

Comment:
Fix the build failure, please.

Also, improve accessibility by using a link text other than "here". Like so, for example.

```suggestion
> You may see the location of a name server [in the Reference section](../../../../reference/designate/index.md).
```